From 2c55337cb0cde08203df321a108409b7e33e6904 Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Wed, 6 Aug 2008 23:13:28 +0100
Subject: [PATCH] fix-build-hash-name-no-colon-separator.patch
 Suggested-by: Carl-Daniel Hailfinger <c-d.hailfinger.devel.2006@gmx.net>

Colons are not very friendly for tar... scp has trouble too but
you can override that by prepending any path like ./ ... still
better to dilute the colon down to a _ avoiding - because it can
appear in branch names we use.

Signed-off-by: Andy Green <andy@openmoko.com>
---
 build |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/build b/build
index 63bb0fc..9fda9c7 100755
--- a/build
+++ b/build
@@ -10,7 +10,7 @@ VERSION=
 if [ -d .git ] ; then
  HEAD=`git show --pretty=oneline | head -n1 | cut -d' ' -f1 | cut -b1-16`
  BRANCH=`git branch | grep ^\* | cut -d' ' -f2`
- VERSION=-$BRANCH:$HEAD
+ VERSION=-$BRANCH\_$HEAD
 fi
 
 if make -j5 ARCH=arm EXTRAVERSION=$VERSION; then
-- 
1.5.6.5

